.github/workflows/ssr-docker.yml

```diff
@@ -0,0 +1,113 @@
+name: SSR Pipeline
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+    types: [ opened, closed ]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ENV: "stg"
+  argocd-file: "values-staging.yaml"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - if: github.ref != 'refs/heads/develop'
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - if: github.ref != 'refs/heads/develop'
+        run: git branch develop origin/develop
+
+      - name: Log in to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+      
+      - name: Extract metadata for docker
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v') ||
+          github.event_name == 'pull_request' && github.event.action == 'opened'
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/superhero-ui-ssr
+          tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=ref,event=tag
+            type=ref,event=pr
```

Comment:
this looks to me like all tags are applied to the same image? is that intentional?

Reply:
They are applied depends on the event

Reply:
we should also check https://github.com/docker/metadata-action#flavor-input, it seems by default `latest` tag is set to any pushed image

Reply:
Yes , do we want this ?

Reply:
I can explicitly set it to false